### DISCUSSION TITLE
Feature suggestion

### DISCUSSION THREAD

**Discussion by jlwinn8** [AlUlkesh/sd_save_intermediate_images#3] [Ideas]
Thank you for making this! This is great. 
I was wondering if it would be possible to display the images on the screen in addition to saving them. It would be awesome to see how different prompts affect the image in real (albeit slow) time. 
thank you!

**Comment by AlUlkesh** [comment]
Do you mean something different than the "normal" preview?
![image](https://user-images.githubusercontent.com/99896447/209233491-34cb773d-b8a8-4d20-9e44-95b159f06a3e.png)

An example would be helpful.

**Comment by jlwinn8** [comment]
Well I learned something new today... lol
Didn't even know that was a setting! Thank you!

**Comment by AlUlkesh** [comment]
Wish I could always help that fast :smile:
